django_lightweight_queue/app_settings.py

```diff
@@ -21,3 +21,7 @@ def setting(suffix, default):
 REDIS_HOST = setting('REDIS_HOST', '127.0.0.1')
 REDIS_PORT = setting('REDIS_PORT', 6379)
 REDIS_PREFIX = setting('REDIS_PREFIX', '')
+
+ENABLE_PROMETHEUS = setting('ENABLE_PROMETHEUS', True)
```

Comment:
We should probably default this to `False` since others using DLQ may not want this.